hearth/sig/lib/hearth/telemetry/otel.rbs

```diff
@@ -0,0 +1,10 @@
+module Hearth
+  module Telemetry
+    class OTelProvider < TelemetryProviderBase
```

Comment:
Should other bases be wrapped here too in case they may change? (i.e. use the whole api as intended)

Reply:
I still don't see a point in wrapping those inner classes.  We control *this* interface here.  If any of the Otel implementations change under the hood, then we can wrap them through this class without breaking anything.

Reply:
Per team discussion last week, we are doing the wrapping approach.